curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/return-a-sorted-array-without-changing-the-original-array.md

```diff
@@ -12,7 +12,7 @@ A side effect of the `sort` method is that it changes the order of the elements
 
 # --instructions--
 
-Use the `sort` method in the `nonMutatingSort` function to sort the elements of an array in ascending order. The function should return a new array, and not mutate the `globalArray` variable.
+Use the `sort` method in the `nonMutatingSort` function to sort the elements of an array in ascending order. The function should return a new array, and not mutate the `globalArray` variable. A callback function must be provided to the `sort` method in order for the tests to pass.
```

Comment:
I'm not sure the additional note is necessary. It reads to me like we are testing that the camper passes a callback function to sort, when instead we are testing that the function returns the correct value.

Reply:
I think you are right (y)